bindata/network/ovn-kubernetes-ipsec/ovn-ipsec.yaml

```diff
@@ -164,6 +164,25 @@ spec:
           }
           trap cleanup SIGTERM
 
+          # Don't start IPsec until ovnkube-node has finished setting up the node
+          counter=0
+          until [ ! -z $(kubectl get pods -l app=ovnkube-node  --field-selector spec.nodeName=${K8S_NODE} -o jsonpath='{.items[0].metadata.name}' 2>/dev/null) ]
```

Comment:
It seems weird to ask kubernetes about this... can't you just do local checks for the expected interface/ovn/ovs state?

Reply:
@danwinship I guess I could but I don't know what to check yet. I want to ensure that the node's networking is set-up before I do anything in case anything changes underneath me. What would be a good thing to check for to ensure that is the case? the cni conf file?

Reply:
I created a new PR for this so I could map a Bz: https://github.com/openshift/cluster-network-operator/pull/929